### PR TITLE
DSND-467: initialzing the project structure

### PR DESCRIPTION
**Pull request by vtondepu**
DSND-467: initialisation of project structure for company-metrics-consumer spring boot application.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+test-unit:
+	@# Help: Run unit tests
+	mvn test -Dskip.integration.tests=true
+
+.PHONY: test-integration
+test-integration:
+	@# Help: Run integration tests
+	mvn integration-test -Dskip.unit.tests=true
+
+,PHONY: run-local
```
Comment (gseregni-ch) [inline Makefile:51]: As identified in other projects this should be .PHONY and not ,PHONY.

File: src/main/java/uk/gov/companieshouse/company/metrics/service/CompanyMetricsApiService.java [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+import uk.gov.companieshouse.api.InternalApiClient;
+import uk.gov.companieshouse.api.model.ApiResponse;
+
+@Service
+public class CompanyMetricsApiService {
+
+    /**
+     * Invoke Insolvency API.
+     */
+    public ApiResponse<?> invokeInsolvencyApi() {
```
Comment (gseregni-ch) [inline src/main/java/uk/gov/companieshouse/company/metrics/service/CompanyMetricsApiService.java:14]: Shouldn't this method be called invokeCompanyMetricsApi? And also the related comment...

**Review by gseregni-ch — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (vtondepu) [reply Makefile]: incorporated the change in file.

File: src/main/java/uk/gov/companieshouse/company/metrics/service/CompanyMetricsApiService.java (hunk 2 above)
Comment (vtondepu) [reply src/main/java/uk/gov/companieshouse/company/metrics/service/CompanyMetricsApiService.java]: good spot. incorporated the change in file.

**Review by gseregni-ch — APPROVED** [review]

File: src/main/java/uk/gov/companieshouse/company/metrics/service/CompanyMetricsApiService.java (hunk 2 above)
Comment (gseregni-ch) [reply src/main/java/uk/gov/companieshouse/company/metrics/service/CompanyMetricsApiService.java]: Thanks

**Pull request merged**